client/src/components/NavBar/index.js

```diff
@@ -1,12 +1,28 @@
 import React from "react";
 import "./NavBar.css";
 import logo from "./logo.png";
+import axios from 'axios';
 
-const NavBar = () => (
+
+function handleSubmit(props){
+
+  const { history } = props;
+  axios
+    .get("/logout")
+    .then(() => {
+      history.push("/");
+    })
+    .catch(error => {
+      const { history } = this.props;
```

Comment:
You defined ```history``` at the beginning of the function, you don't have to define it again